tests/CsvWriteTest.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace Keboola\Csv\Tests;
+
+use Keboola\Csv\CsvWriter;
+use Keboola\Csv\Exception;
+use PHPUnit\Framework\TestCase;
+
+class CsvWriteTest extends TestCase
+{
+    public function testNewFileShouldBeCreated()
+    {
+        self::assertInstanceOf(CsvWriter::class, new CsvWriter(__DIR__ . '/data/non-existent-file.csv'));
+    }
+
+    public function testAccessors()
+    {
+        $csvFile = new CsvWriter(sys_get_temp_dir() . '/test-write.csv');
+        self::assertEquals('test-write.csv', $csvFile->getBasename());
+        self::assertEquals("\"", $csvFile->getEnclosure());
+        self::assertEquals(",", $csvFile->getDelimiter());
+    }
+
+    public function testWrite()
+    {
+        $fileName = sys_get_temp_dir() . DIRECTORY_SEPARATOR . uniqid('csv-test');
+        $csvFile = new CsvWriter($fileName);
+        $rows = [
+            [
+                'col1', 'col2',
+            ],
+            [
+                'line without enclosure', 'second column',
+            ],
+            [
+                'enclosure " in column', 'hello \\',
+            ],
+            [
+                'line with enclosure', 'second column',
+            ],
+            [
+                'column with enclosure ", and comma inside text', 'second column enclosure in text "',
+            ],
+            [
+                "columns with\nnew line", "columns with\ttab",
+            ],
+            [
+                'column with \n \t \\\\', 'second col',
```

Comment:
je tady `'` schválně? Takhle to je skutečně `\n` a ne nový řádek

Reply:
Jo, koukám, že podle testu je to asi expected...

Reply:
ano, je, testuje to, ze se to neinterpretuje